configuration/transitions.md

```diff
@@ -36,7 +36,7 @@ The following transitions are available and executed in order.
 | [death_reporting](#death_reporting) | Updates the deceased status of patients. |
 | conditional_alerts | Executes the configured condition and sends an alert if the condition is met. |
 | [multi_report_alerts](#multi_report_alerts) | Similar to conditional_alerts, with more flexible configuration, including using different form types for the same alert. |
-| [update_notifications](#update_notifications) | **Deprecated** Mutes or unmutes scheduled messages based on configuration. |
+| [update_notifications](#update_notifications) | **Deprecated since 3.2.x** Mutes or unmutes scheduled messages based on configuration. |
```

Comment:
May be clearer to say `Deprecated in 3.2.x` rather than "since" because that could be misinterpreted as deprecated after 3.2.

Reply:
Updated!